src/applications/personalization/dashboard-2/selectors.js

```diff
@@ -1,8 +1,16 @@
 import { toggleValues } from 'platform/site-wide/feature-toggles/selectors';
 import FEATURE_FLAG_NAMES from 'platform/utilities/feature-toggles/featureFlagNames';
 
-export const selectShowDashboard2 = state =>
-  toggleValues(state)[FEATURE_FLAG_NAMES.dashboardShowDashboard2];
+export const selectShowDashboard2 = state => {
+  const LSDashboardVersion = localStorage.getItem('DASHBOARD_VERSION');
+  const LSDashboard1 = LSDashboardVersion === '1';
+  const LSDashboard2 = LSDashboardVersion === '2';
+  const FFDashboard2 = toggleValues(state)[
+    FEATURE_FLAG_NAMES.dashboardShowDashboard2
+  ];
+  const showDashboard2 = !!(LSDashboard2 || (FFDashboard2 && !LSDashboard1));
```

Comment:
Doing `!!()` here because beforehand this selector was returning `null` or `''` when falsey and not `false`